tests/generic_newtype2.rs

```diff
@@ -0,0 +1,44 @@
+extern crate num as num_renamed;
+#[macro_use]
+extern crate num_derive;
+
+use crate::num_renamed::{FromPrimitive};
+use std::ops::Neg;
+use std::num::Wrapping;
+
+#[derive(
+    Debug,
+    Clone,
+    Copy,
+    PartialOrd,
+    ToPrimitive,
+    FromPrimitive,
+    NumOps,
+    NumCast,
+    One,
+    Zero,
+    Num,
+)]
+struct MyThing<T>(Wrapping<T>);
+
+impl<T> PartialEq for MyThing<T> where Wrapping<T>: PartialEq {
```

Comment:
Since `Num` requires `PartialEq`, and the libstd derive causes `PartialEq` to get a `T: PartialEq` bound instead of a `Wrapping<T>: PartialEq` bound, this workaround is required to use the new num derives from this PR with such types.

Reply:
Is there a meaningful difference? The `std` implementation looks like this:

```rust
impl<T> PartialEq<Wrapping<T>> for Wrapping<T>
where
    T: PartialEq<T>,
```
You can't `impl PartialEq for Wrapping<LocalType>`, so it seem like `T: PartialEq` and `Wrapping<T>: PartialEq` should be equivalent.

Reply:
I'm not sure why there's a difference, but there definitely is one. I'll look into it